docs/index.md

```diff
@@ -39,19 +40,19 @@ The rules are under active development, as such the lastest commit on the master
 
 ## Rules
 
-* [rust](rust.md): standard rust rules for building and testing libraries and binaries.
-* [rust_doc](rust_doc.md): rules for generating and testing rust documentation.
-* [rust_proto](rust_proto.md): rules for generating [protobuf](https://developers.google.com/protocol-buffers).
+- [rust](rust.md): standard rust rules for building and testing libraries and binaries.
```

Comment:
Does * to - do something?

Reply:
It doesn't change how it's rendered. [Mardownlint](https://github.com/markdownlint/markdownlint) recommends this this though (or at least is the default style). I recall reading somewhere that this was recommended but cannot remember where.